.travis.yml

```diff
@@ -154,6 +154,16 @@ jobs:
                               -e CC="${CC}"
                               -e CONF="${CONF}"
                               ${DOCKER_NAMESPACE}/travis-odp-lng-ubuntu_16.04 /odp/scripts/ci/distcheck.sh
+                - stage: test
+                  env: TEST=distcheck_nonabi
+                  compiler: gcc
+                  script:
+                          - if [ -z "${DOCKER_NAMESPACE}" ] ; then export DOCKER_NAMESPACE="opendataplane"; fi
+                          - docker run --privileged -i -t
+                              -v `pwd`:/odp --shm-size 8g
+                              -e CC="${CC}"
+                              -e CONF="${CONF}"
+                              ${DOCKER_NAMESPACE}/travis-odp-lng-ubuntu_16.04 /odp/scripts/ci/distcheck_nonabi.sh
```

Comment:
-e CONF="${CONF}"  actually sets up environment in Docker which is passed to run script. I.e. script has to be: ./configure ${CONF}

Reply:
Next version removes separate nonabi.sh and harmonizes CONF usage.